reference/paths/checkout_sessions__delete-session.yaml

```
@@ -0,0 +1,34 @@
+---
+
+operationId: delete-checkout-session
+
+summary: Delete a Checkout Session
+description: Delete a Checkout Session.
```

```suggestion
description: Deletes a Checkout Session.
```

I guess, if we're going for third person in the description, for consistency's sake